Gruntfile.js

```diff
@@ -31,11 +31,45 @@ module.exports = function(grunt) {
           urls: ["http://localhost:8512/tests/SpecRunner.html"]
         }
       }
+    },
+    browserify: {
+      // browserify src/resolver.js -o dist/amd-resolver.js -s amd-resolver --dg false
+      options: {
+        browserifyOptions: {
+          detectGlobals: false,
+          standalone: "amd-resolver"
+        }
+      },
+      dev: {
```

Comment:
I like calling `debug` and `release` for `dev` and `production`.  Production will probably get minification too at some point :)